src/app/contextBroker/contextBroker.cpp

```diff
@@ -628,6 +628,7 @@ PaArgument paArgs[] =
   { "*",      IENTATTRVAL,  IENTATTRVAL_COMPS_V2, IENTATTRVAL_COMPS_WORD,  badVerbGetOnly           }, \
```

Comment:
CNR entry missing (although @crbrox mentioned that probably was a "lost commit", maybe it can be recovered).